src/main/js/main_html5.js

```diff
@@ -681,7 +681,7 @@ require("../../../html5-common/js/utils/environment.js");
       if (_video.textTracks && _video.textTracks.length > 0) {
         var languages = [];
         for (var i = 0; i < _video.textTracks.length; i++) {
-          if (_video.textTracks[i].kind === "captions") {
+          if (_video.textTracks[i].kind === "captions" && !_video.textTracks[i].src === "undefined") {
```

Comment:
This doesn't look to me as a correct check for undefined.  It should be either _video.textTracks[i].src != undefined or typeof _video.textTracks[i].src != "undefined".  I actually tested and it doesn't work right.